hoot-rnd/src/main/cpp/hoot/rnd/conflate/network/OsmNetwork.h

```diff
@@ -0,0 +1,61 @@
+#ifndef OSMNETWORK_H
+#define OSMNETWORK_H
+
+// hoot
+#include <hoot/core/OsmMap.h>
+#include <hoot/rnd/conflate/network/NetworkEdge.h>
+#include <hoot/rnd/conflate/network/NetworkVertex.h>
+
+// Qt
+#include <QList>
+
+namespace hoot
+{
+
+/**
+ * Why not use boost graphs? Well, I used them for the OsmSchema implementation and in my opinion
+ * it is write-once read-never code. While it is likely very efficient it is very difficult to
```

Comment:
ha.  Agree.  I like the capabilities of the boost graphs for the couple of projects I used them on in the past, but found the same thing where the code you end up writing is so hard to read due to the way the library is set up.